riscv_ctg/data/inx.yaml

```diff
@@ -0,0 +1,1532 @@
+# See LICENSE.incore for details
+# IITM Changes
+# This new inx.yaml file handles all of the z*inx extensions
+# =======
+fadd.s:
+  sig:
+    stride: 2
+    sz: 'SIGALIGN'
+  val:
+    stride: 2
+    sz: 'FLEN/8'
+    val_template: "'NAN_BOXED($val,$width,FLEN)'"
+    load_instr: "LREG"
+  xlen: [32,64]
+  isa: 
+    - I_Zfinx
+    - I_Zdinx
+  flen: [32,64]
+  std_op:
+  formattype: 'rformat'
+  rm_val_data: '[7,0,1,2,3,4]'
+  fcsr_data: '[x<<5|y for x,y in itertools.product([0,1,2,3,4],range(0,2**5))]'
+  rs1_op_data: *all_regs
```

Comment:
This is not true in all cases. When `xlen<flen`, the registers are paired and use of odd numbered registers is reserved at this point. It would probably be best to modify this to ensure that the same is captured.

Reply:
This will impact all the `.d` cases. Raised a new [issue](https://github.com/riscv-software-src/riscv-ctg/issues/50) to address this.